sass/components/_popover.scss

```diff
@@ -5,9 +5,11 @@
   position: absolute;
   min-width: 10px;
   opacity: 0;
+  display: none;
```

Comment:
hmm, ok I think that makes sense. The only draw-back is that we may not get the animation but that's fine.